roqoqo-qryd/src/api_backend.rs

```diff
@@ -39,7 +38,7 @@ use std::{thread, time};
 /// This limitation is introduced by design to check the compatability of quantum programs with a model of the QRyd hardware.
 /// For simulations of the QRyd quantum computer use the backend simulator [crate::Backend].
 ///
-#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
```

Comment:
How come `Eq` has been removed?

Reply:
TweezerDevice cannot derive it, consequently I had to cut it.
Additionally, I'm quite sure explicit Eq comparison between APIBackend instances has never been performed. In case something comes up I'll put it back and figure something out for TweezerDevice.